src/app.js

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable quotes */
+/* eslint-disable curly */
+/* eslint-disable comma-dangle */
+/* eslint-disable operator-linebreak */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] You have disabled quite a number of eslint rules. All these can be fixed such no unused vars indicates that there is a declared variable that you are not using anywhere, no use before define means that you are using a variable before defining it. This simply indicates that all these rules do not have to be disabled. Remember, these rules are here to help you write standard accepted code.